MultimodalQnA/docker_compose/intel/cpu/xeon/set_env.sh

```diff
@@ -6,30 +6,50 @@ pushd "../../../../../" > /dev/null
 source .set_env.sh
 popd > /dev/null
 
-export no_proxy=${your_no_proxy}
-export http_proxy=${your_http_proxy}
-export https_proxy=${your_http_proxy}
-export EMBEDDER_PORT=6006
-export MMEI_EMBEDDING_ENDPOINT="http://${host_ip}:$EMBEDDER_PORT/v1/encode"
-export MM_EMBEDDING_PORT_MICROSERVICE=6000
-export ASR_ENDPOINT=http://$host_ip:7066
+export host_ip=$(hostname -I | awk '{print $1}')
+export MM_EMBEDDING_SERVICE_HOST_IP=${host_ip}
+export MM_RETRIEVER_SERVICE_HOST_IP=${host_ip}
+export LVM_SERVICE_HOST_IP=${host_ip}
+export MEGA_SERVICE_HOST_IP=${host_ip}
+
+export no_proxy=${no_proxy}
+export http_proxy=${http_proxy}
+export https_proxy=${http_proxy}
+
+export WHISPER_PORT=7066
+export WHISPER_MODEL="base"
+export ASR_ENDPOINT=http://$host_ip:$WHISPER_PORT
+
```

Comment:
Should we remove this empty line since Whisper and ASR go together? (it would be analogous to how you have LVM and LLAVA server related env vars "grouped")